pkgs/by-name/ri/river-bsp-layout/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "river-bsp-layout";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "areif-dev";
+    repo = "river-bsp-layout";
+    rev = "v${version}";
+    hash = "sha256-LRVZPAS4V5PtrqyOkKUfrZuwLqPZbLoyjn2DPxCFE2o=";
+  };
+
+  cargoHash = "sha256-CtVyRwfIS8R48LUecKXoak+HHB5yNZ5RgguIWOhyFA8=";
+
+  meta = with lib; {
+    description = "https://github.com/areif-dev/river-bsp-layout";
```

```suggestion
    homepage = "https://github.com/areif-dev/river-bsp-layout";
```

Please do also include a description, but the github url would be the homepage.